pandas/core/arrays/masked.py

```diff
@@ -713,7 +713,7 @@ def _hasna(self) -> bool:
         # make this faster by having an optional mask, but not have to change
         # source code using it..
 
-        return cast(bool, self._mask.any())
+        return bool(self._mask.any())
```

Comment:
This looks good, just noting that the implementations for other `_hasna` already have this `bool(...)` where appropriate.